lib/src/layer/marker_layer.dart

```diff
@@ -77,12 +77,14 @@ class Marker {
   final double width;
   final double height;
   final Anchor anchor;
+  final Map<String, dynamic> extras;
```

Comment:
This could just be changed to `dynamic` (thank you Dart :heart:) - and we could decide what kind of data we want to put there ourselves :dart:

```suggestion
  /// Any kind of extra data that you want to pass with the marker
  /// This may be helpful when using clustering
  final dynamic extras;
```